packages/manager/src/features/CloudPulse/shared/CloudPulseRegionSelect.test.tsx

```diff
@@ -48,8 +48,8 @@ describe('CloudPulseRegionSelect', () => {
     const { getByLabelText, getByTestId } = renderWithTheme(
       <CloudPulseRegionSelect {...props} />
     );
-    const { label } = props;
-    expect(getByLabelText(label)).toBeInTheDocument();
+
+    expect(getByLabelText('Region (required)')).toBeInTheDocument();
```

Comment:
This I feel should be the other way around, we can post it in CM team about the POC and get it approved or get a path forward from them. Did we check this with kiran as well?